libraries/AC_Avoidance/AC_Avoid.cpp

```diff
@@ -474,8 +464,11 @@ void AC_Avoid::adjust_velocity_polygon(float kP, float accel_cmss, Vector2f &des
     const float speed = safe_vel.length();
     const Vector2f stopping_point_plus_margin = position_xy + safe_vel*((2.0f + margin_cm + get_stopping_distance(kP, accel_cmss, speed))/speed);
 
-    uint16_t i, j;
-    for (i = 0, j = num_points-1; i < num_points; j = i++) {
+    for (uint16_t i=0; i<num_points; i++) {
```

Comment:
This is a pretty debatable exchange. It's technically 1 less instruction, but it introduces an extra branch which has a larger runtime hit then a single 1 cycle instruction. It's also much wordier like this, but I'd guess that some would find it more readable?

At a minimum please bring in spacing around the `i < num_points`.

Reply:
I find it more readable.. re the spacing, we're a bit all over the place for for loops..  i was also recently wondering (in a different part of the code) if we should have spaces in these situations or not..

Reply:
Definitely spaces for readability! I'd argue pretty forcefully for that one.